### DISCUSSION TITLE
Splitting up a grass field to make it multiple arable fields

### DISCUSSION THREAD

**Discussion by benjen999** [Courseplay/Courseplay_FS22#1889] [Q&A] [answered]
I am curently playing on Attingham Park and I am trying to split F25 into two arable fields and a space for some placables. 
I have tried to draw a custom field  but when I go into the helper menu to slect the field for CP: Field work, it selects the whole of F25 not the section I have maked out. I have attached a screenshot of this.
Am i doing something wrong, or is this just not possible?
![Capture6](https://user-images.githubusercontent.com/112974667/188705707-8966b49b-af7a-4a7d-90b3-fafd851909b7.PNG)

**Comment by Tensuko** [comment, marked as answer]
There is an option in general settings on what CP should prefer, original field or custom field. Try it out :)
